src/core/app/App.ts

```diff
@@ -117,6 +117,8 @@ export default class App {
       }
     ));
 
+    fs.writeFileSync(path.join(this.options.outputDir, 'stories.json'), JSON.stringify(stories));
```

Comment:
This is the change I need - save stories.json, to let any other tool handle the next step.